system/include/emscripten/xmmintrin.h

```diff
@@ -53,17 +53,13 @@ _mm_load_ps(const float *__p)
 static __inline__ __m128 __attribute__((__always_inline__))
 _mm_loadl_pi(__m128 __a, const void /*__m64*/ *__p)
 {
-  // TODO: This actually corresponds to the SIMD.float32x4.loadXY function in
-  // SIMD.js. Use that instead.
-  return (__m128){ ((const float*)__p)[0], ((const float*)__p)[1], __a[2], __a[3] };
+  return __builtin_shufflevector(emscripten_float32x4_loadxy(__p), __a, 0, 1, 4, 5);
```

Comment:
This looks good. The TODO comment this replaces didn't anticipate the shuffle, but even so, this is better than what we had before.